docs/docs/gatsby-cli.md

```diff
@@ -66,121 +66,121 @@ gatsby new
    (Use a different starter)
 ```
 
-See the [Gatsby starters docs](https://www.gatsbyjs.org/docs/gatsby-starters/) for more details.
+अधिक विवरण के लिए [Gatsby शुरुआत डॉक्स](https://www.gatsbyjs.org/docs/gatsby-starters/) देखें।
 
-### `develop`
+### `विकसित करना`
 
-Once you've installed a Gatsby site, go to the root directory of your project and start the development server:
+एक बार जब आप एक Gatsby साइट इनस्टॉल कर लेते हैं, तो अपनी प्रॉजेक्ट की रूट फोल्डर मे जाएँ और डेवेलपमेंट सर्वर शुरू करें:
 
 `gatsby develop`
 
-#### Options
+#### विकल्प
 
-|     Option      | Description                                     |
+|     विकल्प      | विवरण                                     |
 | :-------------: | ----------------------------------------------- |
-| `-H`, `--host`  | Set host. Defaults to localhost                 |
-| `-p`, `--port`  | Set port. Defaults to 8000                      |
-| `-o`, `--open`  | Open the site in your (default) browser for you |
-| `-S`, `--https` | Use HTTPS                                       |
+ `-H`, `--host`  | होस्ट सेट करें। डिफॉल्ट लोकलहोस्ट             |
+| `-p`, `--port`  | पोर्ट सेट करें। डिफॉल्ट ८००० |                      |
+| `-o`, `--open`  | आपके लिए अपने (डिफ़ॉल्ट) ब्राउज़र में साइट खोलता है |
+| `-S`, `--https` | HTTPS का उपयोग करें                                      |
 
-Follow the [Local HTTPS guide](/docs/local-https/)
-to find out how you can set up an HTTPS development server using Gatsby.
+Gatsby का उपयोग करके आप HTTPS डेवलपमेंट सर्वर कैसे सेट कर सकते हैं, यह जानने के लिए [स्थानीय HTTPS गाइड](/docs/local-https/) का उपयोग करें।
 
-#### Preview changes on other devices
+#### अन्य उपकरणों पर प्रीव्यू परिवर्तन
 
-You can use the Gatsby develop command with the host option to access your dev environment on other devices on the same network, run:
+आप एक ही नेटवर्क पर अन्य उपकरणों पर अपने dev एन्वाइरन्मेंट की जानकारीे के लिए होस्ट विकल्प के साथ gatsby develop कमांड का उपयोग कर सकते हैं, चलाएं:
 
 ```shell
 gatsby develop -H 0.0.0.0
 ```
 
-Then the terminal will log information as usual, but will additionally include a URL that you can navigate to from a client on the same network to see how the site renders.
+तब टर्मिनल सामान्य रूप से जानकारी लॉग करेगा, लेकिन इसके अतिरिक्त एक URL भी शामिल होगा जिसे आप साइट क्लाइंट को देखने के लिए उसी नेटवर्क पर क्लाइंट से नेविगेट कर सकते हैं।
 
 ```
-You can now view gatsbyjs.org in the browser.
+अब आप ब्राउज़र में gatsbyjs.org देख सकते हैं।
 ⠀
-  Local:            http://0.0.0.0:8000/
-  On Your Network:  http://192.168.0.212:8000/ // highlight-line
+  स्थानीय:          http://0.0.0.0:8000/
+  अपने नेटवर्क पर:  http://192.168.0.212:8000/ 
 ```
 
-**Note**: you can't visit 0.0.0.0:8000 on Windows (but things will work using either localhost:8000 or the "On Your Network" URL on Windows)
+**ध्यान दें**: आप विंडोज पर 0.0.0.0:8000 पर नहीं जा सकते हैं (लेकिन localhost:8000 पर काम करेंगी या "आपके नेटवर्क" URL पर)
```

Comment:
`विंडोज` => `Windows`

Reply:
this and next one are contradictory(windows or विंडोज)

Reply:
Good catch, we will make it `Windows` everywhere.